flash/ssi_dma/flash_ssi_dma.c

```diff
@@ -67,6 +67,8 @@ uint32_t *expect = (uint32_t *) XIP_NOCACHE_NOALLOC_BASE;
 
 int main() {
     stdio_init_all();
+    
+    sleep_ms(2500);
```

Comment:
If you want to "wait for minicom to connect" it's much better to use `PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS` - see https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_stdio_usb/include/pico/stdio_usb.h#L57 and https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_stdio_usb/CMakeLists.txt#L21

Reply:
Thank you @lurch, I modified CMakeLists.txt with the next lines and it worked:
```

if (NOT DEFINED PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS)
    set(PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS 3000)
endif()
```

Making the sleep_ms(2500) redundant.

Reply:
Thank you @lurch, I modified CMakeLists.txt with the next lines and it worked:
```

if (NOT DEFINED PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS)
    set(PICO_STDIO_USB_CONNECT_WAIT_TIMEOUT_MS 3000)
endif()
```

Making the sleep_ms(2500) redundant.